plugins/inputs/interrupts/README.md

```diff
@@ -8,6 +8,10 @@ The interrupts plugin gathers metrics about IRQs from `/proc/interrupts` and `/p
   ## To filter which IRQs to collect, make use of tagpass / tagdrop, i.e.
   # [inputs.interrupts.tagdrop]
     # irq = [ "NET_RX", "TASKLET" ]
+  #
+  # To report cpus as tags instead of fields use cpus_as_tags
+  # cpus_as_tags = false
```

Comment:
i am using telegraf version 1.9.2 and this is not working.
Error parsing **file.conf**, line 76: field corresponding to `cpu_as_tag' is not defined in `*interrupts.Interrupts'

Reply:
You need at least version 1.10.  We try to mark the milestone a feature will be released in on the pull request, you should be able to see it on the right sidebar.